tests/grammar/vyper.lark

```diff
@@ -0,0 +1,144 @@
+// Vyper grammar for Lark
+// Adapter from Lark repo. https://github.com/lark-parser/lark/blob/master/examples/python3.lark
+// Adapted from: https://docs.python.org/3/reference/grammar.html
+// Adapted by: Erez Shinan
+
+// Start symbols for the grammar:
+//       single_input is a single interactive statement;
+//       file_input is a module or sequence of commands read from an input file;
+//       eval_input is the input for the eval() functions.
+// NB: compound_stmt in single_input is followed by extra NEWLINE!
+single_input: _NEWLINE | simple_stmt | compound_stmt _NEWLINE
+file_input: (_NEWLINE | stmt)*
+eval_input: testlist _NEWLINE*
+
+decorator: "@" decorator_names _NEWLINE
+         | "@" decorator_with_args_names [ "(" [arguments] ")" ] _NEWLINE
+decorators: decorator+
+decorated: decorators (funcdef)
+
+funcdef: "def" NAME "(" parameters? ")" ["->" test] ":" suite
+
+parameters: paramvalue ("," paramvalue)* [","]
+
+?paramvalue: typedparam ["=" test]
+?typedparam: NAME [":" test]
+
+varargslist: (vfpdef ["=" test] ("," vfpdef ["=" test])* ["," [ "*" [vfpdef] ("," vfpdef ["=" test])* ["," ["**" vfpdef [","]]] | "**" vfpdef [","]]]
+  | "*" [vfpdef] ("," vfpdef ["=" test])* ["," ["**" vfpdef [","]]]
+  | "**" vfpdef [","])
+
+vfpdef: NAME
+
+?stmt: simple_stmt | compound_stmt
+?simple_stmt: small_stmt (small_stmt)* _NEWLINE
+?small_stmt: (expr_stmt | pass_stmt | flow_stmt | import_stmt | assert_stmt)
+?expr_stmt: testlist_star_expr (annassign | augassign (testlist)
+         | ("=" (testlist_star_expr))*)
+annassign: ":" test ["=" test]
+?testlist_star_expr: (test) ("," (test))* [","]
+!augassign: ("+=" | "-=" | "*=" | "/=" | "**=")
+// For normal and annotated assignments, additional restrictions enforced by the interpreter
+pass_stmt: "pass"
+?flow_stmt: break_stmt | continue_stmt | return_stmt | raise_stmt
+break_stmt: "break"
+continue_stmt: "continue"
+return_stmt: "return" [testlist]
+raise_stmt: "raise" [test ["from" test]]
+import_stmt: import_name | import_from
+import_name: "import" dotted_as_names
+import_from: "from" (dots? dotted_name | dots) "import" ("*" | "(" import_as_names ")" | import_as_names)
+!dots: "."+
```

Comment:
Shouldn't this be either 1 or 2, and not 1+?

Reply:
yep; done.